keras/layers/convolutional.py

```diff
@@ -1245,6 +1252,262 @@ def get_config(self):
+            `channels_last` corresponds to inputs with shape
+            `(batch, height, width, channels)` while `channels_first`
+            corresponds to inputs with shape
+            `(batch, channels, height, width)`.
+            It defaults to the `image_data_format` value found in your
+            Keras config file at `~/.keras/keras.json`.
+            If you never set it, then it will be "channels_last".
+        depth_multiplier: The number of depthwise convolution output channels
+            for each input channel.
+            The total number of depthwise convolution output
+            channels will be equal to `filterss_in * depth_multiplier`.
+        activation: Activation function to use
+            (see [activations](../activations.md)).
+            If you don't specify anything, no activation is applied
+            (ie. "linear" activation: `a(x) = x`).
+        use_bias: Boolean, whether the layer uses a bias vector.
+        depthwise_initializer: Initializer for the depthwise kernel matrix
+            (see [initializers](../initializers.md)).
+        pointwise_initializer: Initializer for the pointwise kernel matrix
+            (see [initializers](../initializers.md)).
+        bias_initializer: Initializer for the bias vector
+            (see [initializers](../initializers.md)).
+        depthwise_regularizer: Regularizer function applied to
+            the depthwise kernel matrix
+            (see [regularizer](../regularizers.md)).
+        pointwise_regularizer: Regularizer function applied to
+            the pointwise kernel matrix
+            (see [regularizer](../regularizers.md)).
+        bias_regularizer: Regularizer function applied to the bias vector
+            (see [regularizer](../regularizers.md)).
+        activity_regularizer: Regularizer function applied to
+            the output of the layer (its "activation").
+            (see [regularizer](../regularizers.md)).
+        depthwise_constraint: Constraint function applied to
+            the depthwise kernel matrix
+            (see [constraints](../constraints.md)).
+        pointwise_constraint: Constraint function applied to
+            the pointwise kernel matrix
+            (see [constraints](../constraints.md)).
+        bias_constraint: Constraint function applied to the bias vector
+            (see [constraints](../constraints.md)).
+
+    # Input shape
+        3D tensor with shape:
+        `(batch, channels, steps)` if data_format='channels_first'
+        or 3D tensor with shape:
+        `(batch, steps, channels)` if data_format='channels_last'.
+
+    # Output shape
+        3D tensor with shape:
+        `(batch, filters, new_steps)` if data_format='channels_first'
+        or 3D tensor with shape:
+        `(batch, new_steps, filters)` if data_format='channels_last'.
+        `new_steps` values might have changed due to padding or strides.
+    """
+
+    @interfaces.legacy_separable_conv2d_support
```

Comment:
This is a new layer so no API conversion decorator is necessary.

Reply:
Updated.